websites/O/OtakuEyes/metadata.json

```diff
@@ -0,0 +1,27 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "297055773714808833",
+		"name": "Yù"
+	},
+	"service": "OtakuEyes",
+	"description": {
+		"en": "My own platform for watching animes",
+		"fr": "Mon site pour regarder des animes"
+	},
+	"url": "192.168.0.7",
```

Comment:
From your project's README, it looks like you can visit this project in many places, including `localhost`, `127.0.0.1`, etc. Consider using an array of URLs or a regular expression.